.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,90 @@
+name: Bug report
+description: Create a report to help us improve FOSSBilling
+title: "[Bug]"
+labels: [bug]
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      Please ensure that there is no existing issue for your bug, and fill out the title before continuing.
+      **Important: DO NOT include any private/sensitive information such as email addresses, passwords, IP addresses, or hostnames.**
+
+- type: textarea
+  id: describe
+  attributes:
+    label: Describe the bug
+    description: A clear and concise description of what the bug is.
+    placeholder: Tell us what you're facing! Include as much detail as possible.
+  validations:
+    required: true
+
+- type: textarea
+  id: reproduce
+  attributes:
+    label: How to reproduce
+    description: What steps did you take when the bug occured? Help us reproduce it!
+    placeholder: |
+      Steps to reproduce the behavior:
+      1. Go to '...'
+      2. Click on '....'
+      3. Scroll down to '....'
+      4. See error
+  validations:
+    required: true
+
+- type: textarea
+  id: expected
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen.
+    placeholder: What is the expected behavior of the action you took? Help us understand!
+  validations:
+    required: true
+
+- type: textarea
+  id: screenshots
+  attributes:
+    label: Screenshots
+    description: If applicable, add screenshots to help explain your problem.
+    placeholder: Use the "Attach an image or video" button or link your images manually using the Markdown syntax, if you're familiar with it.
+
+- type: dropdown
+  id: device
+  attributes:
+    label: Device
+    description: Please select the type of device you encountered the issue in.
+    multiple: true
+    options:
+      - Desktop
+      - Smartphone
+  validations:
+    required: true
+ 
+- type: textarea
+  id: info
+  attributes:
+    label: Information
+    description: Please complete the following information.
+    placeholder: |
+      - Device: [e.g. iPhone 6] (only applicable for smartphones)
+      - OS: [e.g. Fedora 37]
+      - Browser [e.g. chrome, safari]
+      - Version [e.g. 22]
+  validations:
+    required: true
+
+- type: input
```

Comment:
Changed the priority and added an optional module field.

Reply:
Issues for modules ( Eg: Proxmox, SolusVM and CentovaCast) Should have different Bug reports then the the main "Fossbilling" repos.

That why I suggest we should move the bug templates to  /FOSSBilling/FOSSBilling/.github/

I hope at least Github is smart enough that it will prefer templates in the repo of the general .github repo